examples/microphone-speech-to-text.py

```diff
@@ -37,6 +37,7 @@
 # initialize speech to text service
 authenticator = IAMAuthenticator('your_api_key')
 speech_to_text = SpeechToTextV1(authenticator=authenticator)
+speech_to_text.set_service_url('https://stream.watsonplatform.net/speech-to-text/api')
```

Comment:
I think this will fail because we run our examples automatically and this parameter gets set from an environment variable